code/modules/clothing/gloves/boxing.dm

```diff
@@ -23,3 +23,23 @@
 	icon_state = "boxingyellow"
 	item_state = "boxingyellow"
 	species_fit = list(VOX_SHAPED)
+
+/obj/item/clothing/gloves/boxing/attackby(obj/W, mob/user)
+	..()
+
+	if(istype(W, /obj/item/bear_hands))
+		visible_message("\the [user] attaches \the [W] to \the [src]")
+		new/obj/item/clothing/gloves/boxing/bear(get_turf(src))
+		user.drop_item(W, force_drop = 1)
+		user.drop_item(src, force_drop = 1)
```

Comment:
Why would I be able to attach bear hands to a pair of gloves if they're stuck to my hand?